node/src/cli.rs

```diff
@@ -70,6 +70,14 @@ pub struct ExportGenesisStateCommand {
 	/// The name of the chain for that the genesis state should be exported.
 	#[structopt(long)]
 	pub chain: Option<String>,
+
+	/// Number of accounts to be funded in the genesis in dev mode (default 10)
+	#[structopt(long, default_value = "10")]
+	pub accounts: u32,
+
+	/// Mnemonic from which we can derive funded accounts in the genesis
+	#[structopt(long)]
+	pub mnemonic: Option<String>,
```

Comment:
I think the correct place to add these two new cli flags is to the `build-spec` subcommand. It is when you're defining a new chain spec that you have the opportunity the fund different accounts at genesis.